pkg/frontend/v2/frontend_test.go

```diff
@@ -125,6 +132,7 @@ func TestFrontendBasicWorkflow(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, int32(200), resp.Code)
 	require.Equal(t, []byte(body), resp.Body)
+	t.Cleanup(f.StopAsync)
```

Comment:
Why do we call `f.StopAsync` here, but also `services.StopAndAwaitTerminated(context.Background(), f)` (which stops the same service and waits until it has stopped)?

(I'd keep previous code, calling `_ = services.StopAndAwaitTerminated(context.Background(), f)` in the Cleanup function, and remove this line.)

Reply:
oh, I committed this by mistake, sorry